src/plugins/explore/public/components/query_panel/components/footer/types.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export enum ResultStatus {
+  UNINITIALIZED = 'uninitialized',
+  LOADING = 'loading', // initial data load
+  READY = 'ready', // results came back
+  NO_RESULTS = 'none', // no results came back
```

Comment:
should the enum value be the same like
```suggestion
  NO_RESULTS = 'NO_RESULTS', // no results came back
```

Reply:
same behaviour  as existing error type for discover.